apps/frontend/src/app/pages/company/home/home.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
```

Comment:
Just noticed this problem in the responsivity when the page is 518px wide in my monitor
![image](https://github.com/Venture-Miner/wagademy/assets/60355496/de15eaae-5d86-4457-b5c1-0c7a54f5cfd3)

Reply:
Try opening developer tools and shrink slowly the page in your device toolbar so you can notice

Reply:
Right, Bia! I adjusted now :)